tests/integration/test_relation.py

```diff
@@ -227,3 +251,37 @@ def get_ingress_annotation():
         ingress_annotations["nginx.ingress.kubernetes.io/enable-owasp-modsecurity-crs"] == "true"
     )
     assert ingress_annotations["nginx.ingress.kubernetes.io/modsecurity-snippet"]
+
+
+@pytest.mark.usefixtures("build_and_deploy", "setup_new_hostname_and_port")
+@pytest.mark.parametrize(
+    "anycharm_update_ingress_config",
+    [
+        {
+            "rewrite-target": "/foo",
```

Comment:
Would be worth testing if changing back to no value, the rewrite-target default is "/" as expected?

Reply:
I added a new test case for the default rewrite-target value, thanks!